public/scripts/project/lock.js

```diff
@@ -0,0 +1,75 @@
+define(function(require) {
+  var fs = Bramble.getFileSystem();
+  var sh = new fs.Shell();
+  var LOCK_KEY = 'bramble-lock';
+  var EXPIRY = 2000; // MS
```

Comment:
When you have a time value, always include the time units in the name: `LOCK_EXPIRY_MS`